WordPress/Classes/Models/Blog/Blog+SelfHosted.swift

```diff
@@ -17,7 +18,7 @@ extension Blog {
 
     static func createRestApiBlog(
         with details: WpApiApplicationPasswordDetails,
-        in contextManager: ContextManager,
+        in contextManager: CoreDataStackSwift,
```

Comment:
In this PR, I decided not to rename the `contextManager` variable that now have `CoreDataStackSwift` to keep the diff smaller. 

We might want to do it later on, but it does not seem urgent.